File: modules/jackson/src/main/scala/com/github/eikek/calev/jackson/CalevModule.java
package com.github.eikek.calev.jackson;

import com.fasterxml.jackson.databind.module.SimpleModule;
import com.github.eikek.calev.CalEvent;

public final class CalevModule extends SimpleModule {
	private static final long serialVersionUID = 1L;

	public CalevModule() {
		super(PackageVersion.VERSION);

		addDeserializer(CalEvent.class, new CalEventDeserializer());
		addSerializer(CalEvent.class, new CalEventSerializer());
	}

}

File: modules/jackson/src/main/scala/com/github/eikek/calev/jackson/CalevModule.scala
package com.github.eikek.calev.jackson

import com.fasterxml.jackson.databind.module.SimpleModule
import com.github.eikek.calev.CalEvent

@SerialVersionUID(1L)
final class CalevModule() extends SimpleModule(PackageVersion.VERSION) {
  addDeserializer(classOf[CalEvent], new CalEventDeserializer)
  addSerializer(classOf[CalEvent], new CalEventSerializer)
}
